src/Symfony/Component/Messenger/Bridge/Doctrine/Transport/Connection.php

```diff
@@ -127,32 +127,52 @@ public static function buildConfiguration(#[\SensitiveParameter] string $dsn, ar
      */
     public function send(string $body, array $headers, int $delay = 0): string
     {
+
         $now = new \DateTimeImmutable('UTC');
-        $availableAt = $now->modify(sprintf('%+d seconds', $delay / 1000));
+        $availableAt = $now->modify(sprintf('+%d seconds', $delay / 1000));
```

Comment:
why did you change this line?
it was updated quite recently, see https://github.com/symfony/symfony/pull/53187